tests/test_forum.py

```diff
@@ -0,0 +1,286 @@
+        assert topic.icon_custom_emoji_id == TEST_TOPIC_EMOJI_ID
+
+    def test_to_dict(self, forum_group_id, topic):
+        topic_dict = topic.to_dict()
+
+        assert isinstance(topic_dict, dict)
+        assert topic_dict["message_thread_id"] == forum_group_id
+        assert topic_dict["name"] == TEST_TOPIC_NAME
+        assert topic_dict["icon_color"] == TEST_TOPIC_ICON_COLOR
+        assert topic_dict["icon_custom_emoji_id"] == TEST_TOPIC_EMOJI_ID
+
+    def test_equality(self, forum_group_id):
+        a = ForumTopic(
+            message_thread_id=forum_group_id,
+            name=TEST_TOPIC_NAME,
+            icon_color=TEST_TOPIC_ICON_COLOR,
+        )
+        b = ForumTopic(
+            message_thread_id=forum_group_id,
+            name=TEST_TOPIC_NAME,
+            icon_color=TEST_TOPIC_ICON_COLOR,
+            icon_custom_emoji_id=TEST_TOPIC_EMOJI_ID,
+        )
+        c = ForumTopic(
+            message_thread_id=forum_group_id,
+            name=f"{TEST_TOPIC_NAME}!",
+            icon_color=TEST_TOPIC_ICON_COLOR,
+        )
+        d = ForumTopic(
+            message_thread_id=forum_group_id + 1,
+            name=TEST_TOPIC_NAME,
+            icon_color=TEST_TOPIC_ICON_COLOR,
+        )
+        e = ForumTopic(
+            message_thread_id=forum_group_id,
+            name=TEST_TOPIC_NAME,
+            icon_color=0xFFD67E,
+        )
+
+        assert a == b
+        assert hash(a) == hash(b)
+
+        assert a != c
+        assert hash(a) != hash(c)
+
+        assert a != d
+        assert hash(a) != hash(d)
+
+        assert a != e
+        assert hash(a) != hash(e)
+
+    async def test_get_forum_topic_icon_stickers(self, bot):
+        # TODO this fails
+        # we expect the first to stay as it is. This might change in the future.
+        # If we have to fix this test too often maybe just checking it is set to "something"
+        # is enough.
+        emoji_sticker_list = await bot.get_forum_topic_icon_stickers()
+        print(emoji_sticker_list[0].emoji)
+        assert emoji_sticker_list[0].emoji == "📰"
+        assert emoji_sticker_list[0].height == 512
+        assert emoji_sticker_list[0].width == 512
+        assert emoji_sticker_list[0].is_animated
+        assert not emoji_sticker_list[0].is_video
+        assert emoji_sticker_list[0].set_name == "Topics"
+        assert emoji_sticker_list[0].type == Sticker.CUSTOM_EMOJI
+        assert emoji_sticker_list[0].custom_emoji_id == "5420143492263320958"
+        assert emoji_sticker_list[0].thumb.width == 128
+        assert emoji_sticker_list[0].thumb.height == 128
+        assert emoji_sticker_list[0].thumb.file_size == 4036
+        assert emoji_sticker_list[0].thumb.file_unique_id == "AQADfh0AAso3OEty"
+        assert emoji_sticker_list[0].file_size == 57126
+        assert emoji_sticker_list[0].file_unique_id == "AgADfh0AAso3OEs"
+
+    # we sadly do not have access to a test group right now so we only test params right now
```

Comment:
we do now so let's use that

Reply:
These two tests (this comment and [previous one](https://github.com/python-telegram-bot/python-telegram-bot/pull/3356#discussion_r1021672564)) were created by @Poolitzer if I'm not mistaken (I had copy-pasted this file from a different branch). I would tweak them, of course, if @Poolitzer didn't change these lines in a different branch.